snapcraft/internal/meta/_manifest.py

```diff
@@ -18,16 +18,20 @@
 import os
 import json
 from collections import OrderedDict
-from typing import Any, Dict  # noqa: F401
+from typing import Any, Dict, TYPE_CHECKING
```

Comment:
Ah interesting, I was looking for a constant like this one to avoid importing packages just for type checking.